.env.template

```
@@ -1,12 +1,12 @@
-APP_ENV=
+APP_ENV=development
 APP_PORT=3000
-DB_HOST=
-DB_PORT=
-DB_NAME=
-DB_USER=
-DB_PASS=
+DB_HOST=localhost
```

```suggestion
DB_HOST=0.0.0.0
```
I think this might be better. This will make sure it will work in docker-compose as well. :thinking: 

Also these will make sure `docker-compose` compatibility 
```
DB_NAME=example_db
DB_USER=root
DB_PASS=example
```